source/administration/open-source-components.rst

```diff
@@ -45,6 +46,7 @@ Mobile
 Redux
 ------------------------------
 
+ - Mattermost Enterprise Edition v5.18.0 - `View Open Source Components <https://github.com/mattermost/mattermost-redux/blob/release-5.18/NOTICE.txt>`_.
```

Comment:
@amyblais I forget, why do we reference `Enterprise Edition` in this case?

Reply:
@jasonblais I'm not sure but I can remove it.

Reply:
@amyblais 0/5. We could also say `Mattermost Redux v5.18.0`, etc. But I'm not sure if there's a contractual reason for having Enterprise Edition, since the binary we ship includes webapp, server and redux. Melissa Warner might know